src/test/resources/META-INF/mods.toml

```diff
@@ -172,5 +172,6 @@ license="LGPL v2.1"
     modId="part_entity_test"
 [[mods]]
     modId="capabilities_test"
-
```

Comment:
Don't need to remove the new line at the end of the file.

Reply:
did not think of that. I added it back